e2e_samples/parking_sensors/README.md

```diff
@@ -244,67 +244,65 @@ More resources:
 *NOTE: This deployment was tested using WSL 2 (Ubuntu 18.04) and Debian GNU/Linux 9.9 (stretch)*
 
 1. **Initial Setup**
+   - **Fork** this repository into a new Github repo.
+   - Set the following **required** environment variables:
+      - **TENANT_ID** - an Entra ID. Used to login to a specific tenant with Azure CLI
+      - **AZURE_SUBSCRIPTION_ID** - Azure subscription id to use to deploy resources. Default: default azure subscription.
+      - **AZDO_PROJECT** - Target Azure DevOps project where Azure Pipelines and Variable groups will be deployed
+      - **AZDO_ORGANIZATION_URL** - Target Azure DevOps Organization of Azure DevOps project in this form `https://dev.azure.com/<organization>/`. (ei. https://dev.azure.com/my_azdo_org/). Must be in the same tenant as $TENANT_ID
+      - **GITHUB_REPO** - Name of your forked github repo in this form `<my_github_handle>/<repo>`. (ei. "devlace/mdw-dataops-import")
+      - **GITHUB_PAT_TOKEN** - a Github PAT token. Generate them [here](https://github.com/settings/tokens). This requires "repo" scope.
+
+      Optionally, set the following environment variables:
+      - **AZURE_LOCATION** - Azure location to deploy resources. *Default*: `westus`.
+      - **DEPLOYMENT_ID** - string appended to all resource names. This is to ensure uniqueness of azure resource names. *Default*: random five character string.
+      - **AZDO_PIPELINES_BRANCH_NAME** - git branch where Azure DevOps pipelines definitions are retrieved from. *Default*: main.
+      - **AZURESQL_SERVER_PASSWORD** - Password of the SQL Server instance. *Default*: random string.
+
+   - If you are using **dev container**, follow the below steps:
+      - Rename `.envtemplate` under ".devcontainer" folder to `.env` and update the values as mentioned above instead of setting those as environment variables.
+      - Open the project inside the vscode dev container (see details [here](docs/devcontainer.md)).
+
+   - To further customize the solution, set parameters in `arm.parameters` files located in the `infrastructure` folder.
+      - To enable Observability and Monitoring components through code(Observability-as-code), please set enable_monitoring parameter to true in  `arm.parameters` files located in the `infrastructure` folder. This will deploy log analytics workspace to collect monitoring data from key resources, setup an Azure dashboards to monitor key metrics and configure alerts for ADF pipelines.
    - Ensure that:
       - You are logged in to the Azure CLI. To login, run
 
         ```bash
-        az login
+        az config set core.login_experience_v2=off
+        az login --tenant $TENANT_ID
```

Comment:
In task #685 we are removing Az Login from the individual scripts and instead including it in the pre-requisites documentation. We should sync up on this as I have a PR open to make things consistent. It has not been pushed yet.  PR #880